components/NavBar.tsx

```diff
@@ -0,0 +1,44 @@
+import Image from "next/image";
+import Link from "next/link";
+import { Lobster } from "next/font/google";
+
+const lobster = Lobster({ weight: "400", subsets: ["latin"] });
+
+export default function NavBar() {
+  // return <p>test</p>;
+
+  return (
+    <header
+      className={`flex justify-between items-center border-b-2 border-white p-4 ${lobster.className}`}
+    >
+      <Link href="/">
+        <svg
```

Comment:
I'd make a generic `Icon` component that accepts a `type` prop:

```tsx
<Icon type="home" />
```

Reply:
Created an Icon component but did not insert a type cuz I'm not sure why I need it. Also I can't define a type prop as a string apparently.

Reply:
Inserted type prop in Icon component. Moved Home and Search icons to Icon component